packs/orion/actions/ncm_config_download.py

```diff
@@ -27,6 +27,10 @@ def run(self, node, platform, configs):
         results = {}
         node_ids = []
 
+        # Split into an Array, as enum and array type does not play nice
+        # in the WebUI.
+        configs = configs.split(",")
```

Comment:
Can we make this parameter an array without an enum attribute in the action metadata?